metricbeat/module/ceph/mgr_cluster_disk/_meta/fields.yml

```diff
@@ -0,0 +1,21 @@
+- name: mgr_cluster_disk
+  type: group
+  description: >
+    mgr_cluster_disk
+  release: beta
+  fields:
+    - name: available.bytes
+      type: long
+      description: >
+        Available bytes of the cluster
+      format: bytes
+    - name: total.bytes
+      type: long
+      description: >
+        Total bytes of the cluster
+      format: bytes
+    - name: used.bytes
+      type: long
+      description: >
+        Used bytes of the cluster
+      format: bytes
```

Comment:
Could we reuse the same field names as we are using with old modules? This way the transition will be easier and we will be able to reuse dashboards and so on.

Reply:
Right now all field names are as much identical as possible. The difference is in metric set names:

```yaml
- name: cluster_disk
  type: group
  description: >
    cluster_disk
  release: ga
  fields:
    - name: available.bytes
      type: long
      description: >
        Available bytes of the cluster
      format: bytes
    - name: total.bytes
      type: long
      description: >
        Total bytes of the cluster
      format: bytes
    - name: used.bytes
      type: long
      description: >
        Used bytes of the cluster
      format: bytes
```

and

```yaml
- name: mgr_cluster_disk
  type: group
  description: >
    mgr_cluster_disk
  release: beta
  fields:
    - name: available.bytes
      type: long
      description: >
        Available bytes of the cluster
      format: bytes
    - name: total.bytes
      type: long
      description: >
        Total bytes of the cluster
      format: bytes
    - name: used.bytes
      type: long
      description: >
        Used bytes of the cluster
      format: bytes

```